### PR TITLE
Update to work with V1 of ruby lib

### PR DESCRIPTION
**Pull request by jackfranklin**
@petehamilton can you do the honours? Tested all locally and it's glorious.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -128,7 +127,7 @@
                     when "EUR" then "€"
                     end
   @price = "#{currency_symbol}#{"%.2f" % PACKAGE_PRICES[package][currency]}"
-  @first_payment_date = subscription.upcoming_payments.first[:charge_date]
+  @first_payment_date = subscription.upcoming_payments.first["charge_date"]
```
Comment (jackfranklin) [inline app.rb]: not sure what's caused the change here, and why this worked before...

File: Gemfile [hunk 2]
```diff
@@ -1,10 +1,10 @@
 source "https://rubygems.org"
 
+gem "pry"
```
Comment (jackfranklin) [inline Gemfile]: useful to leave in I think, I make enough mistakes frequently that it's guaranteed I'll need this LOTS

File: app.rb (hunk 1 above)
Comment (petehamilton) [inline app.rb]: I sort of like symbols... When did they stop working? :(

Comment (jackfranklin) [inline app.rb]: Some time between 0.2 and 1.0, but not something I did on purpose...

Comment (jackfranklin) [inline app.rb]: Having looked through the diff, I've no idea on this. Will look into it as v1.0.1...

**Comment by petehamilton** [comment]
:+1:

**Pull request merged**